test/CMakeLists.txt

```diff
@@ -123,6 +122,13 @@ if(NOT (MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906 OR MIOPEN_TEST_GFX908 OR MIOPEN
         set(MIOPEN_TEST_GPU_DETECTION_FAILED TRUE)
     endif()
 endif()
+
+set_var_to_condition(MIOPEN_LIBMLIR_SUPPORTS_GFX103X_DEFAULT ${MIOPEN_USE_MLIR} AND (${rocMLIR_VERSION_FLAT} GREATER 100000000))
```

Comment:
@atamazov maybe we should check if `rocMLIR_VERSION_FLAT` is set here? It is only set when `MIOPEN_USE_MLIR` is true

Reply:
Thanks for informing. I missed this case, will fix tomorrow.

Reply:
@junliume I thought that CMake employs _short-circuiting_ for AND and OR, but this is not so, and `${MIOPEN_USE_MLIR} AND ...` does not break evaluation when MIOPEN_USE_MLIR is Off.

Reply:
Resolved.